.github/actions/workflow-vars/action.yml

```diff
@@ -43,11 +61,11 @@ runs:
       # https://github.com/orgs/community/discussions/40291
       GITHUB_WORKFLOW_JOB_ID=$(rand_gen | cut -c 1-6)
 
-      echo "Env vars:"
-
       # Workflow.
       echo "GITHUB_WORKFLOW_JOB_ID=$GITHUB_WORKFLOW_JOB_ID" | tee -a $GITHUB_ENV
+      echo "GITHUB_WORKFLOW_JOB_ID=$GITHUB_WORKFLOW_JOB_ID" >> "$GITHUB_OUTPUT"
```

Comment:
since you are already using `tee` you can save yourself the duplication.

```
echo "GITHUB_WORKFLOW_JOB_ID=$GITHUB_WORKFLOW_JOB_ID" | tee -a "$GITHUB_ENV" "$GITHUB_OUTPUT"
```